### ISSUE TITLE
Bug: no-duplicate-keyframe-selectors only checks the first selector in a selector list

### ISSUE THREAD

**Issue by minseonkkim** [eslint/css#506]
### Environment

ESLint version: v10.7.0
@eslint/css version: v1.4.0
Node version: v22.15.0
npm version: v11.12.1
Operating System: Windows 11

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```js
import css from "@eslint/css";

export default [
	{
		files: ["**/*.css"],
		plugins: { css },
		language: "css/css",
		rules: {
			"css/no-duplicate-keyframe-selectors": "error",
		},
	},
];
```
</details>

```css
/* Case 1: duplicate between a selector list and a single selector — NOT reported */
@keyframes a {
	0%, 50% { top: 0; }
	50% { top: 1px; }
}

/* Case 2: same duplicate, other order — NOT reported */
@keyframes b {
	50% { top: 0; }
	0%, 50% { top: 1px; }
}

/* Case 3: duplicate within a single selector list — NOT reported */
@keyframes c {
	0%, 0% { top: 0; }
}

/* Control: exact duplicate — correctly reported */
@keyframes e {
	50% { top: 0; }
	50% { top: 1px; }
}
```

### What did you expect to happen?

All three keyframe blocks (`a`, `b`, and `c`) should report a duplicate
keyframe selector, since the offset `50%` (or `0%`) appears more than once
within the same `@keyframes` block — whether the duplicate occurs between a
selector list and a single selector, or within one selector list.

For comparison, stylelint's `keyframe-block-no-duplicate-selectors` reports
duplicates inside selector lists as well.

### What actually happened?

Only the control case in `@keyframes e` is reported:

```
/path/to/repro.css
  21:2  error  Unexpected duplicate selector '50%' found within keyframe block  css/no-duplicate-keyframe-selectors

✖ 1 problem (1 error, 0 warnings)
```

Cases 1–3 produce no warnings.

I believe the cause is in `src/rules/no-duplicate-keyframe-selectors.js` —
the rule only reads the first selector of each prelude:

```js
const selector = node.prelude.children[0].children[0];
```

so any selector after a comma in a `SelectorList` is never recorded nor
checked against the `seen` map. Iterating over all of
`node.prelude.children` fixes all three cases while keeping the existing 28
tests passing (verified locally).

Related question, happy to handle in the same PR if desired: the current
tests include `0% { } 0.0% { }` as a *valid* (non-duplicate) case, but these
are numerically the same offset. Should percentages be compared numerically
(e.g. `Number(selector.value)`) so `50%` and `50.0%` count as duplicates?

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/stackblitz-starters-stunqgmr

### Participation

- [x] I am willing to submit a pull request for this issue.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

**Comment by Pixel998** [comment]
This is a duplicate of #459, so I'm closing this issue.
